### ISSUE TITLE
VirusTotal trojan detect ratio 25/69

### ISSUE THREAD

**Issue by skrzacikus** [JakubSteplowski/SpotifyLyricsNET#7]
25 from 69 engines in VirusTotal detects trojans in exe file
https://www.virustotal.com/gui/file/113065dbb715edaa86f515544ce67aac323d8a1276c709ff3c0e8999a475ad0c/detection

**Comment by JakubSteplowski** [comment]
Hi!

This means they work correctly, nice for them!
But that also means that 44 don't...

The reason is pretty simple, my software is using a similar structure of a trojan virus, but other than that it's harmless.

The main .exe contains another .exe (that's why it's seen as a trojan) which is nothing else than the "helper" that is used to make the "Launch with Spotify" feature possible, you can find its source in this repository (/Source/C# - Helper/).
That software works in the background and starts at the startup of Windows to detect if you opened Spotify (if the spotify process is present) and if you did (and the "Launch with Spotify" feature is enabled, which means a file "launch_flag" is present in "/Documents/Spotify Lyrics .NET") it launches the lyrics software. As simple as that.

The reason why I've chose this solution is the fact I release this app also on the Microsoft Store, and making a UWP app that runs in background and starts with Windows is pretty difficult and for some versions of Windows 10 even impossible.
An easier, and still great in my opinion, solution is this one, using a separate (and even lighter than main .exe) program just for that.

I hope it's clear enough, if it's not or you have other questions, don't hesitate to ask them.

Best Regards,
Jakub

**Comment by skrzacikus** [comment]
Yes, it's clear for me, bot not for those anti-malware/virus/trojan engines :)
You can try to report it as false positive.

**Comment by mohitarora24** [comment]
Okay. In this case, could we get a version without this feature? or a special binary for non-UWP app?

**Comment by JakubSteplowski** [comment]
> Okay. In this case, could we get a version without this feature? or a special binary for non-UWP app?

The binary of the non-UWP version is already present here on github:
https://github.com/JakubSteplowski/SpotifyLyricsNET/releases

If it gets detected by your antivirus as false postive add it to the exceptions.

**Comment by mohitarora24** [comment]
Yeah, I downloaded from there and it got detected as Virus. Actually, it is a company laptop I'm installing it on and I can't modify AntiVirus settings (add exception or disable it).

**Comment by JakubSteplowski** [comment]
> Yeah, I downloaded from there and it got detected as Virus. Actually, it is a company laptop I'm installing it on and I can't modify AntiVirus settings (add exception or disable it).

I will think about a different version, in the meantime use v1.4.1:
https://github.com/JakubSteplowski/SpotifyLyricsNET/releases/tag/v1.4.1

It has a slightly different UI, no Launch with Spotify feature, but the other features are the same.

**Comment by AGonLen** [comment]
I was coming to say exactly the same....  My antivirus detected it (and deleted the .exe accordingly) as a thread!

![2020-05-28 094521](https://user-images.githubusercontent.com/24992203/83127181-25759900-a0da-11ea-9fd0-d89d8474e7ba.jpg)

**Comment by JakubSteplowski** [comment]
> I was coming to say exactly the same.... My antivirus detected it (and deleted the .exe accordingly) as a thread!
> 
> You are saying that using the UWP version (including latest releases) would not be detected as a Trojan?
> 
> ![2020-05-28 094521](https://user-images.githubusercontent.com/24992203/83127181-25759900-a0da-11ea-9fd0-d89d8474e7ba.jpg)

I can't assure you that, unfortunately.
The easiest way is to add the program to the exceptions, or if you don't trust me, just use v1.4.1 as 
I said above.
https://github.com/JakubSteplowski/SpotifyLyricsNET/releases/tag/v1.4.1
But you lose many fixes and improvements.

v2 should be fine, but can't tell already when I will release it.

**Comment by AGonLen** [comment]
Thanks for letting me know. I'm using your version, but looking forward to v2, as soon as you can release it, no worries!
